tests/test_basic.py

```diff
@@ -22,11 +22,15 @@ def datatype_table(loop, cursor, table_cleanup):
 
 @pytest.mark.run_loop
 async def test_datatypes(connection, cursor, datatype_table):
+    encoding = connection.charset
+    if encoding == 'utf8mb4':
```

Comment:
a bit confused about this one, can you please elaborate?

Reply:
PyMYSQL has set the default charset to utf8mb4, that isnt a valid encoding in python terms, so as the variable is later used to decode a string, I converted it to a known python encoding.